### ISSUE TITLE
[ACCESSIBILITY] Visited Recently: change navy text on navy background to white

### ISSUE THREAD

**Issue by vburgett** [rokwire/illinois-app#5443]
1. Go to Home > Sections > Visited Recently
2. See how the "Clear All" link in the upper righthand corner of the panel is navy with an orange underline on the navy background.

**The link text should be white with an orange underline.** 

(Did this link change to navy after we added the same "Clear All" text to the Assistant panel?)

v7.2.25 (prod, iPhone)
v7.3.10 (dev, Android)

<img width="1080" height="2400" alt="Image" src="https://github.com/user-attachments/assets/b944e201-644d-46cf-a2f1-8e5b2b9d0718" />

**Comment by vburgett** [comment]
Confirmed as fixed.
